policyengine_us/variables/gov/states/az/tax/income/credits/property_tax_credit/az_property_tax_credit_eligible.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class az_property_tax_credit_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Arizona Property Tax Credit"
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.az.tax.income.credits.property_tax
+
+        age = person("age", period)
+        age_eligible = age >= p.min_age
+
+        ssi = person("ssi", period)
+        ssi_qualifies = ssi > 0
+
+        property_tax = person("real_estate_taxes", period)
```

Comment:
This is part of the credit itself so can remove from this formula

Reply:
it is an eligibility criteria to pay rent / have property taxes, would we not create an eligibility file in this case as both are also part of the calculation

Reply:
Ok I see it's in the law too, even if redundant, so let's keep it. But use add to check in one statement